README.md

```diff
@@ -155,24 +154,24 @@ Install the package using:
 
 ### Other Linux
 
-On others you'll need to compile it yourself using the (simple) instructions
+On others, compile the browser using the instructions
```

Comment:
Maybe "DB4S" instead of "the browser"?

Reply:
I put DB4S in this spot. As I was reading having no indication of what was being compiled confused me as a reader, so it may help readers of this document.